sunpy/net/helioviewer.py

```diff
@@ -207,19 +266,22 @@ def download_png(self, date, image_scale, layers, directory=None,
         >>> from sunpy.net.helioviewer import HelioviewerClient
         >>> import datetime
         >>> hv = HelioviewerClient()  # doctest: +REMOTE_DATA
-        >>> file = hv.download_png('2012/07/16 10:08:00', 2.4, "[SDO,AIA,AIA,171,1,100]", x0=0, y0=0, width=1024, height=1024)   # doctest: +REMOTE_DATA
-        >>> file = hv.download_png('2012/07/16 10:08:00', 4.8, "[SDO,AIA,AIA,171,1,100],[SOHO,LASCO,C2,white-light,1,100]", x1=-2800, x2=2800, y1=-2800, y2=2800)   # doctest: +REMOTE_DATA
-        >>> file = hv.download_jp2(datetime.datetime.now(), observatory='SDO', instrument='HMI', detector='HMI', measurement='continuum')   # doctest: +REMOTE_DATA
+        >>> file = hv.download_png('2012/07/16 10:08:00', 2.4, "[SDO,AIA,AIA,171,1,100]", False, x0=0, y0=0, width=1024, height=1024)   # doctest: +REMOTE_DATA
+        >>> file = hv.download_png('2012/07/16 10:08:00', 4.8, "[SDO,AIA,AIA,171,1,100],[SOHO,LASCO,C2,white-light,1,100]", True, x1=-2800, x2=2800, y1=-2800, y2=2800)   # doctest: +REMOTE_DATA
         """
         params = {
             "action": "takeScreenshot",
             "date": self._format_date(date),
             "imageScale": image_scale,
             "layers": layers,
-            "display": True
+            "eventLabels": eventLabels,
+            "display": True,
+            "events": None,
+            "scale": False,
+            "watermark": False
```

Comment:
You've hard coded these, they should take the input from the user.

Reply:
I just their default values. User can enter its values if they want but if they don't enter anything then as discussed before I gave the default values to those parameters

Reply:
But it won't update to the user values.

Reply:
Oh, ok I will change that